internal/blockchain/v2/reactor.go

```diff
@@ -415,9 +427,9 @@ func (r *BlockchainReactor) demux(events <-chan Event) {
 			case pcBlockProcessed:
 				r.setSyncHeight(event.height)
 				if r.syncHeight%100 == 0 {
-					lastRate = 0.9*lastRate + 0.1*(100/time.Since(lastHundred).Seconds())
+					r.lastSyncRate = 0.9*r.lastSyncRate + 0.1*(100/time.Since(lastHundred).Seconds())
```

Comment:
Same here as above with having an initial clause to handle when r.lastSyncRate is 0